### ISSUE TITLE
Player as USSR having relased GER in the begining ,the western germany cannot be released.

### ISSUE THREAD

**Issue by yf22** [simsulla/nwomod#336]
It should be forbidden

**Comment by yf22** [comment]
May be a false alarm.As the German capital is 574.Waiting for detail.

**Comment by simsulla** [comment]
Need to create event with conditions
```
fire_only_once = yes
trigger = { 
  tag = DDR
  NOT = { year = 1956 }
  exists = GER
  government = proletarian_dictatorship
  NOT = { exists = DDR }
}
option = {
  change_tag_no_core_switch = DDR
}
```
In events/occupation_of_germany.txt
Would you feel up to making such event. Pick an unused event id like 8204085

**Comment by yf22** [comment]
Notice that change_tag_no_core_switch does not mean what it says as the command change_tag will not switch any core automactically.
Change_tag_no_core_switch is used to switch the player to another county from the current one without any changes in territories.For example,it is used in the USA civil war events in HOD and the round-the-world-in-80-days events of story model in my own mod DAW.

**Comment by simsulla** [comment]
Fixed by #338
